packages/gen-ai-hub/src/orchestration/api/schema/completion-post-request.ts

```diff
@@ -13,8 +13,4 @@ export type CompletionPostRequest = {
   orchestration_config: OrchestrationConfig;
   input_params: Record<string, InputParamsEntry>;
   messages_history?: ChatMessages;
-  /**
-   * Whether to return module results, which returns detailed information about each module execution. Be careful with the content of the individual module results as they may contain unchecked user or model content.
-   */
-  return_module_results?: boolean;
```

Comment:
~[q] Should we not support users providing this option?~ Nvm, I see the option was removed with the spec update.